packages/cli/src/workflows/workflow-execution.service.ts

```diff
@@ -97,7 +97,7 @@ export class WorkflowExecutionService {
 		}: WorkflowRequest.ManualRunPayload,
 		user: User,
 		pushRef?: string,
-		partialExecutionVersion?: string,
+		partialExecutionVersion: 1 | 2 = 1,
```

Comment:
This union is repeated a few times, maybe worth it to have `type PartialExecutionVersion = 1 | 2`